src/mlmc/quantity_types.py

```diff
@@ -75,6 +75,9 @@ def _make_getitem_op(self, chunk, key):
         """
         return QType.keep_dims(chunk[key])
```

Comment:
Considering the file name, we should possibly introduce all classes into `mlmc` module via. `mlmc.__init__.py`.

Reply:
#164